docs/rapid-pro-rsdl.md

```diff
@@ -1,92 +1,90 @@
 ---
 id: rsdl
-title: Rapid Schema Definition Language (RSDL)
-sidebar_label: Rapid Schema Definition
+title: RAPID Schema Definition Language (RSDL)
+sidebar_label: RAPID Schema Definition
 ---
 
-RapidPro allows to design your services using the Rapid Service Definition Language (RSDL).
+RAPID allows to design your services using the RAPID Service Definition Language (RSDL).
 RSDL is an expression language for capturing services, entities, operations and various expressions in human-readable format.
 
 RSDL syntax is very similar to many popular schema formats like GraphQL,
 bringing simplicity and lowering amount of time required to learn it. 
 RSDL is compatible with GraphQL syntax which gives numerous benefits:
 
--   All existing IDE plugins for GraphQL syntax highlighting will work with Rapid format.
--   Numerous of tools for validating schema complexity, detecting changes can be also used with Rapid
+-   All existing IDE plugins for GraphQL syntax highlighting will work with RAPID format.
+-   Numerous of tools for validating schema complexity, detecting changes can be also used with RAPID
 -   Developers can use various tools to generate their models directly from database definitions or even code.
--   Simplified migration path from GraphQL to Rapid by reusing parts of the same schema.
+-   Simplified migration path from GraphQL to RAPID by reusing parts of the same schema.
 
 > NOTE: RSDL can be compiled into CSDL JSON and CSDL XML formats using a subset of the features of the OData specification
 
 ## Building your first type
 
-Developers who would like to represent their datamodel will need to create new Rapid schema file,
+Developers who would like to represent their datamodel will need to create new RAPID schema file,
 for example `MyRapidPro.graphql`. At minimum schema will require at least one Entity Type.
 Entity type is being build by specifying `type` keyword and listing all it's fields like follows:
 
-
 ```graphql
-type Person {
-    UserName: String! @RapidID
-    FirstName: String
-    LastName: String
-    MiddleName: String
-    Age: BigInt
+type company {
```

Comment:
Any reason for small letter?

Reply:
This is the example service we are using for the intro page: https://jetsons.azurewebsites.net/$metadata